examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -25,6 +25,16 @@ using namespace ::chip;
 
 CHIP_ERROR ModelCommand::RunCommand()
 {
+
+    if (IsGroupId(mNodeId))
+    {
+        ChipLogProgress(chipTool, "Sending command to group 0x%" PRIx16 " on Fabric Index 0x%" PRIx16, GroupIdFromNodeId(mNodeId),
+                        static_cast<FabricIndex>(mEndPointId));
+
+        return SendGroupCommand(GroupIdFromNodeId(mNodeId), static_cast<FabricIndex>(mEndPointId),
```

Comment:
Why are we forcing consumers to pass in a fabric index (which they have no good way to know)?  We should be getting the fabric index from `CurrentCommissioner()`.

Reply:
Done